disruption_generator/listener/alistener.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+
+import asyncssh
+import re
+import time
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class AlistenerException(Exception):
+    pass
+
+
+class Alistener(object):
+    def __init__(self, hostname):
+        self.hostname = hostname
+        self.files = []
+
+    async def run_client(self, filepath, expression, timeout):
+        async with await asyncssh.connect(self.hostname) as conn:
+            logger.debug("Connected to %s", self.hostname)
+            stdin, stdout, stderr = await conn.open_session("tail -f %s" % filepath)
```

Comment:
tail -f will put all the log to the memory, and if we have file rotate it will not work,
maybe try:
tail -F <fileName> | grep --line-buffered  <pattern> -A 3 -B 5
Or tail -F <fileName> | grep  <pattern> -C 5

Reply:
The receive buffers are cleared everytime we read from stdout.
https://groups.google.com/d/topic/asyncssh-users/i1ouhyekxt8/discussion